crates/figma_import/src/reflection.rs

```diff
@@ -177,6 +177,9 @@ pub fn registry() -> serde_reflection::Result<serde_reflection::Registry> {
     tracer
```

Comment:
Let me know if I should remove it or keep it..
For the figma_schema::FigmaColor. I am not quite sure why some need in reflection and some not.

Reply:
Yes, remove the FigmaColor one. Reflection basically defines the types that the serde code generates Java code for. Primarily that is the types used in DesignDefinition. The goal is to keep FigmaColor out of DesignDefinition so it should be removed from reflection.rs.

Reply:
Done.